Foundation/NSLog.swift

```diff
@@ -41,3 +41,11 @@ public func NSLog(_ format: String, _ args: CVarArg...) {
         NSLogv(format, $0) 
     }
 }
+
+// hook for System.out.println()
+#if os(Android)
+public var ANLog = {
```

Comment:
Maybe `AndroidLog` then, since `AN` is a two-letter prefix? But does this hook have to live in NSLog.swift?